spec/system/oai_pmh_spec.rb

```diff
@@ -78,9 +78,8 @@
     end
   end
 
-  # should not create an error, as long as it's possible in DB
   describe "work with no representative" do
-    let!(:work_with_no_representative) { create(:work, :with_complete_metadata, title: "no representative", published: true) }
+    let!(:work_with_no_representative) { build(:work, :published, :with_complete_metadata, title: "no representative", members: [], representative: nil).tap {|w| w.save!(validate: false)} }
```

Comment:
Might not need the complicated `save.validate(false)` here either, now that we got rid of the model validation?

Reply:
Nope -- it's still necessary here.

Reply:
Very confusing! Can you help explain to me why it's needed here, but not above in child_count_display_fetcher_spec.rb that also does a `let(:item) { build(:public_work, members: []) }`? What's the difference?

Reply:
It's because `build` doesn't save the item to the DB. The `save.validate(false)` has the side-effect of turning the `build` into a `create`, which in turn means that the work is then found in the DB by `WorkOaiProvider`.
I'll change the `build` to `create` -- then I can remove the `save`.

Reply:
OK, right, you don't actually need the `validate: false`, then? And therefore there's no reason to use `build` instead of `create`?  This pattern of `build... save(validate:false)` was to do the equivalent of a `create` while skipping the validation.  So if just `create` works here, that's simpler!

Reply:
Nope... you can't use just `create`. OAI is looking for records in the database, so if you just use `build`, it won't find your work!

Reply:
Yes! Create works great, but you also *need* it so that the OAI provider can find the record in the database.